src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/entity/OffenceScheduleMapping.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.manageoffencesapi.entity
+
+import org.hibernate.annotations.Fetch
+import org.hibernate.annotations.FetchMode
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Entity
+@Table
+data class OffenceScheduleMapping(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long = -1,
+  @ManyToOne
+  @Fetch(FetchMode.JOIN)
+  val scheduleParagraph: ScheduleParagraph,
+  val lineReference: String? = "",
```

Comment:
Sorry don't know the context of how these are used.  If they can be null is there a reason for assigning an empty string to them?

Reply:
no there isn't - copy and paste mistake!